scripting/get5_pgsqlstats.sp

```diff
@@ -0,0 +1,345 @@
+/**
+ * =============================================================================
+ * Get5 PostgreSQL stats
+ * Copyright (C) 2016. Sean Lewis.  All rights reserved.
+ * =============================================================================
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cstrike>
+#include <sourcemod>
+
+#include "get5/version.sp"
+#include "include/get5.inc"
+#include "include/logdebug.inc"
+
+#include "get5/util.sp"
+
+#pragma semicolon 1
+#pragma newdecls required
+
+Database db = null;
+char queryBuffer[2048];
+
+int g_MatchID = -1;
```

Comment:
Don't duplicate `g_MatchID` here. Use the `GetMatchId` method on the events from the forwards: